README.md

```diff
@@ -33,11 +33,10 @@ We want full OOP.
 
 Oh, and we don't want to write C code to make the code run faster.
 
-Status
+Project Status
 ------
 
-* The project is in alpha stage: we are still tweaking the language and standard library.
-* The compiler is written in Crystal.
+Crystal is still under heavy development. There can be breaking changes but we're trying to keep those as minimum as possible.
```

Comment:
> "The compiler uses LLVM 4.0 to compile to native code."? Because I think it could be interesting to know which LLVM version Crystal is currently using.

I'd skip it, since it's a moving target, mine is built with LLVM 5.0.1 for instance.

Reply:
"those as minimum as possible" should be "them as minimal as possible"